shell/src/main/java/alluxio/cli/GetConf.java

```diff
@@ -114,13 +145,21 @@ public static int getConf(String... args) {
         } else {
           if (cmd.hasOption(UNIT_OPTION_NAME)) {
             String arg = cmd.getOptionValue(UNIT_OPTION_NAME).toUpperCase();
-            Unit unit;
             try {
-              unit = Unit.valueOf(arg);
-              System.out.println(Configuration.getBytes(key) / unit.getValue());
-            } catch (IllegalArgumentException e) {
-              printHelp(String.format("%s is not a valid unit", arg));
-              return 1;
+              ByteUnit byteUnit;
+              byteUnit = ByteUnit.valueOf(arg);
+              System.out.println(Configuration.getBytes(key) / byteUnit.getValue());
+              break;
+            } catch (Exception e) {
+              try {
```

Comment:
it is not ideal to nest `try-catch` blocks inside another outer `try-catch` block.
Can you just make the first catch ignore the exception, and have the second `try-catch` in parallel?

Thanks